data/js/controls.js

```diff
@@ -730,11 +730,11 @@ var addToHTML = function(data) {
   } else {
     //We are adding to an existing panel so we only need the HTML for the element
     var parent = $("#id" + data.parentControl);
-    parent.append(elementHTML(data.type, data.id, data.value, elementStyle));
+    parent.append(elementHTML(data.type, data.id, data.value, data.label, elementStyle));
```

Comment:
Oops, my bad. I'm tacking this on the existing PR to get it merged in ASAP.

Reply:
👍